experimental/Schemes/CoherentSheaves.jl

```diff
@@ -689,8 +689,8 @@ function twisting_sheaf(IP::AbsProjectiveScheme{<:Field}, d::Int)
     (UU, VV) = glueing_domains(G)
     h_U = complement_equation(UU)
     h_V = complement_equation(VV)
-    MG[(U, V)] = (d>= 0 ? OO(VV)(h_V^d) : (inv(OO(VV)(h_V))^(-d)))*identity_matrix(OO(VV), 1)
-    MG[(V, U)] = (d>= 0 ? OO(UU)(h_U^d) : (inv(OO(UU)(h_U))^(-d)))*identity_matrix(OO(UU), 1)
+    MG[(U, V)] = (d>= 0 ? (x->OO(VV)(x, check=false))(h_V^d) : (inv((x->OO(VV)(x, check=false))(h_V))^(-d)))*identity_matrix(OO(VV), 1)
```

Comment:
you could just create a matrix directly instead of multiplying by the identity?

Reply:
I do not really see what you are doing here, sorry.

Reply:
Neither do I, in either the old or new version of that code. So perhaps that can be improved, by adding a comment, and/or splitting this overly clever computation into multiple lines? E.g. instead of
```
MG[(U, V)] = (d>= 0 ? (x->OO(VV)(x, check=false))(h_V^d) : (inv((x->OO(VV)(x, check=false))(h_V))^(-d)))*identity_matrix(OO(VV), 1)
```
how about
```
if d >= 0
  tmp = (x->OO(VV)(x, check=false))(h_V^d)
else
  tmp = inv((x->OO(VV)(x, check=false))(h_V))^(-d)
end
MG[(U, V)] = tmp*identity_matrix(OO(VV), 1)
```

This is already *much* easier to read and understand, and to me immediately shows further potential for improvement, resp. makes me ask questions; e.g.
- that last line should probably be using `diagonal_matrix` (please read its docstring);
- the auxiliary function `x->OO(VV)(x, check=false)` seems completly redundant? I.e. why can't `(x->OO(VV)(x, check=false))(h_V^d)` be replaced by `OO(VV)(h_V^d, check=false)` ?
- in one place first power then apply  `OO(VV)`, in the other it's the other way around. Intentional? Does it make a difference?